src/modules/wreck/lua.d/spectrum.lua

```diff
@@ -0,0 +1,47 @@
+-- Set environment specific to spectrum_mpi (derived from openmpi)
```

Comment:
Commit message body should be filled in.  Maybe something like

> Add mpi "personality" for IBM spectrum MPI, enabled by user with wreckrun -o mpi-spectrum.
> Note that this plugin assumes MPI is installed in /opt/ibm/spectrum_mpi.
> It also disables PAMI, the spectrum enhanced collectives due to their dependency on the RM providing
> a PMIx server.  See #1382 for further details.
> It also sets the soft stack limit to a value the MPI runtime seems to require.
>
> See #1382 for more details.
>
> Fixes #1584

Reply:
Will do. Sorry for the sloppiness there.

Reply:
Done